cmd/make.go

```diff
@@ -199,11 +259,33 @@ func runTestcases() {
 
 func clean() {
 	gopBinPath := detectGopBinPath()
-	if checkPathExist(gopBinPath, true) {
-		if err := os.RemoveAll(gopBinPath); err != nil {
-			println(err.Error())
+	goBinPath := detectGoBinPath()
+
+	if !checkPathExist(gopBinPath, true) {
+		return
+	}
+
+	gopBinFiles, err := os.ReadDir(gopBinPath)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	// Clean links
+	for _, file := range gopBinFiles {
```

Comment:
don't clean links. only delete links when we build success.

Reply:
Now, only clean when uninstall.

Reply:
Now, only clean when uninstall.